junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/TestSuitesModel.java

```diff
@@ -36,10 +36,34 @@
     @XmlElement(name = "testsuite")
     private List<TestSuiteModel> testsuite;
 
+    private Boolean hasMissingAttributes() {
+        return failures == null || time == null || errors == null || tests == null;
+    }
+
     public void postProcess() {
         uniqueID = UUID.randomUUID().toString();
 
-        if ((failures != null && Integer.parseInt(failures) > 0) || (errors != null && Integer.parseInt(errors) > 0)) {
+        if (hasMissingAttributes()) {
+            int failuresCount = 0;
+            int errorsCount = 0;
+            Double totalTime = 0.0;
+            for (TestSuiteModel suite : testsuite) {
+                if (suite.getOverallStatus().equals(Constants.FAILED)) {
+                    failuresCount++;
+                } else if (suite.getOverallStatus().equals(Constants.ERRORED)) {
+                    errorsCount++;
+                }
+                totalTime += Double.parseDouble(suite.getTime());
+            }
+
+            // update fields if necessary
+            if (failures == null) failures = Integer.toString(failuresCount);
+            if (tests == null) tests = Integer.toString(testsuite.size());
+            if (time == null) time = Double.toString(totalTime);
+            if (errors == null) errors = Integer.toString(errorsCount);
```

Comment:
i know it is not mandatory to have brackets for single line statement, but please add them on a new row, and not on the same line, from my point of view it looks more readable